cross-chain/arbitrum/test/utils/upgrades.ts

```diff
@@ -0,0 +1,31 @@
+import { deployments, ethers, upgrades } from "hardhat"
```

Comment:
Why aren't we using `@keep-network/hardhat-helpers` for upgrades?

https://github.com/keep-network/hardhat-helpers/blob/main/src/upgrades.ts

Reply:
No particular reason. Do you have an example where we use `@keep-network/hardhat-helpers` in tests? If this is not a lot of redoing, I can try to replace it.